src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/payment/PaymentApiClient.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.client.payment;
```

Comment:
Could the client be one like https://github.com/hmcts/fact-api/blob/master/src/main/java/uk/gov/hmcts/dts/fact/config/FeignConfiguration.java 

Then you can autowire it in and then use it 

I think feigns used on the other ones too, can double check and let me know if not. Food for thought

Reply:
other clients in this repo are not using feign client such as src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/cdam/CdamApiClient.java but can be refactered in a future ticket.